src/Microsoft.Extensions.Configuration.EnvironmentVariables/EnvironmentVariablesConfigurationProvider.cs

```diff
@@ -20,15 +23,24 @@ public class EnvironmentVariablesConfigurationProvider : ConfigurationProvider
 
         private readonly string _prefix;
 
+        /// <summary>
+        /// Initializes a new instance.
+        /// </summary>
         public EnvironmentVariablesConfigurationProvider() : this(string.Empty)
         { }
 
-
+        /// <summary>
+        /// Initializes a new instance with the specified prefix.
+        /// </summary>
+        /// <param name="prefix">The prefix used to filter the keys.</param>
```

Comment:
Thinking about this one more, it doesn't filter the keys bout the environment variables...

`The prefix used to filter the environment variables.`